packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -787,36 +812,44 @@ export default class SalesforceCRMService implements CRM {
     const conn = await this.conn;
     const appOptions = this.getAppOptions();
 
-    if (!appOptions?.createEventOn) {
+    this.log.info(`Changing owner for record ${id}`);
```

Comment:
We're adding a lot more logging to the `checkRecordOwnerNameFromRecordId` method instead of just returning